x-pack/plugins/reporting/index.js

```diff
@@ -146,7 +147,8 @@ export const reporting = (kibana) => {
       validateConfig(config, message => server.log(['reporting', 'warning'], message));
       logConfiguration(config, message => server.log(['reporting', 'debug'], message));
 
-      const xpackMainPlugin = server.plugins.xpack_main;
+      const { xpack_main: xpackMainPlugin, monitoring: monitoringPlugin } = server.plugins;
```

Comment:
This needs testing on what happens when Monitoring is disabled but Reporting is enabled. cc @pickypg